### ISSUE TITLE
[Question]Improving accuracy of model and avoiding false detections

### ISSUE THREAD

**Issue by mriamnobody** [Deci-AI/super-gradients#1027]
I love this repository. It is powerful and easy to implement. The main feature of this repository for me is that it takes multiple camera streams as input without much complication. @Louis-Dupont helped me a lot to achieve and build my mini home DIY project. This is my first project as a beginner in computer vision and YOLO. After implementing and modifying the reference script provided by Louis, I observed that at night the object which was properly detected was being falsely detected as human. Even in the daylight the model falsely detects some objects incorrectly.

Is there a way to fine-tune and tell the model that it is not what it has detected? Also How I improve the human detection accuracy at night? As of now, I have not faced any major false detections of humans in daylight. I have a lot of images from the cameras in my house with humans in the frame. In those frames, the humans/persons are inside a blue bounded box. Will this help me to fine-tune it?

I'm a beginner. Please forgive me for my noobiness. Thank you.

Below is my script:

```
import cv2
import time
import asyncio
import logging
from super_gradients.training import models
from super_gradients.common.object_names import Models
from telegram import Bot
from telegram.error import TelegramError

# Set up logging
logging.basicConfig(filename='app.log', 
                    filemode='w', 
                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s',
                    level=logging.INFO)
logger = logging.getLogger(__name__)

TOKEN = 'YOUR_BOT_TOKEN'
CHAT_ID = 'YOUR_CHAT_ID'

bot = Bot(token=TOKEN)

async def send_message(text):
    try:
        await bot.send_message(chat_id=CHAT_ID, text=text)
    except TelegramError as e:
        logger.error("Failed to send message through Telegram with error: %s", e)

# Create a loop to run the async function in
loop = asyncio.get_event_loop()
loop.run_until_complete(send_message("Detection Program Started"))

def release_resources():
    logger.info("Releasing video capture objects and closing windows")
    cap1.release()
    cap2.release()
    cap3.release()
    cap4.release()
    cap5.release()
    cv2.destroyAllWindows()

start_time = time.time()
logger.info("Script start time: %s", start_time)

try:
    logger.info("Starting to load model")
    model = models.get(Models.YOLO_NAS_L, pretrained_weights="coco").cuda()
    logger.info("Detection Model loaded successfully")
    loop.run_until_complete(send_message("Detection Model loaded successfully"))
except Exception as e:
    logger.error("Detection Model loading failed with error: %s", e)
    loop.run_until_complete(send_message("Model loading failed with error: " + str(e)))

try:
    cap1 = cv2.VideoCapture('camstream1')
    cap2 = cv2.VideoCapture('camstream2')
    cap3 = cv2.VideoCapture('camstream3')
    cap4 = cv2.VideoCapture('camstream4')
    cap5 = cv2.VideoCapture('camstream5')
except Exception as e:
    logger.error("VideoCapture initialization failed with error: %s", e)
    loop.run_until_complete(send_message("VideoCapture initialization failed with error: " + str(e)))

while True:
    try:
        captures = [(cap1, 'cam_name1'), (cap2, 'cam_name2'), (cap3, 'cam_name3'), (cap4, 'cam_name4'), (cap5, 'cam_name5')]
        frames = []
        logger.info("Reading frames from cameras")

        for cap, camera_name in captures:
            ret, frame = cap.read()
            if not ret:
                loop.run_until_complete(send_message(f"Failed to read frames from camera {camera_name}"))
            else:
                frames.append(frame)

        logger.info("Predicting frames")
        predictions = model.predict(frames)

        for i, (predicted_frame, (_, camera_name)) in enumerate(zip(predictions, captures)):
            cv2.imshow(camera_name, predicted_frame.draw())
            
        if cv2.waitKey(1) & 0xFF == ord("q"):
            break
    except Exception as e:
        logger.error("An error occurred: %s", e)
        loop.run_until_complete(send_message("An error occurred: " + str(e)))
        time.sleep(5) # optional: wait before trying again
        continue

release_resources()

end_time = time.time()
logger.info("Script end time: %s", end_time)

execution_time = end_time - start_time
logger.info("Script executed in: %s seconds", execution_time)
print(f"Script executed in: {execution_time} seconds")
loop.run_until_complete(send_message(f"Script executed in: {execution_time} seconds"))
```

**Comment by Louis-Dupont** [comment]
If you want to night accuracy, you can experiment with changing the confidence threshold (`model.predict(..., conf=...)`) on inference. This is not the ideal solution, but it would be very quick to test.

The most robust approach would be to [fine-tune](https://colab.research.google.com/drive/1q0RmeVRzLwRXW-h9dPFSOchwJkThUy6d) your model on a dataset including night images. If you don't have such a dataset, you can always play with lightning of the images to improve model robustness on low lightning images. You can explore [Roboflow Universe](https://roboflow.com/universe) hub, they have lot's of ready-to-use datasets.
